routes/profile/addUserProfile.js

```diff
@@ -0,0 +1,33 @@
+/*
+Purpose of API: This API will update the user's profile, and save their changes in the SQL database. 
+*/
+
+const express = require('express');
+const router = express.Router();
+const mysql = require('mysql');
+const config = require('../../config.js');
+
+router.post('/api/addUserProfile', (req, res) => {
```

Comment:
nitpicky, but this API should be called `createUserProfile`. Rename all project references as well. You can do this with `Crtl` + `Shift` + `H`

Reply:
also rename the file too